docs/dpdk.md

```diff
@@ -0,0 +1,194 @@
+# Kube-OVN with OVS-DPDK
+
+This document describes how to run Kube-OVN with OVS-DPDK.
+
+
+## Prerequest
+- Kubernetes >= 1.11
+- Docker >= 1.12.6
+- OS: CentOS 7.5/7.6/7.7, Ubuntu 16.04/18.04
+
+
+## To Install
```

Comment:
we should add a section about how to setup dpdk on the host

Reply:
OVS must be on the same environment as DPDK. If I move OVS-DPDK to the host then OVN also needs to move to the host because OVN RPMs depend on OVS RPMs. We agreed this does not fit well with Kube-OVN and so I focused my efforts on containerizing OVS-DPDK. The Docker image now contains DPDK, OVS and OVN. Nothing needs to be preinstalled on the host.

Reply:
Do we need setup hugepage, cpu-mask and other system config on host?

Reply:
Good catch on the hugepages, these need to be setup on the host. I’ll document how to do this.

CPU mask can be used but not necessary. I purposely didn’t mention it here because it gets quite complex. If using CPU mask, you should also use something like [CMK](https://github.com/intel/CPU-Manager-for-Kubernetes). Configuration here is complex as I said, and it’s very difficult to write instructions that are generic for every user. I can mention that ovs-vsctl has cpu-mask option and I can link to the CMK project. But I think instructions are out of the scope of this document.